database/migrations/postgres/1720428454051313_fk_deployments.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE deployments
+    ADD COLUMN transformerEslId INTEGER
```

Comment:
there should not be any changes in existing migrations. Please create new migrations instead.